### DISCUSSION TITLE
[A-L] (2023/24) Foglio 5 - Esercizio 1

### DISCUSSION THREAD

**Discussion by Elia-Belli** [sapienzastudentsnetwork/calcolo-delle-probabilita#164] [Esercizi - Bertini] [answered]
![Screenshot 2023-10-30 134103](https://github.com/sapienzastudentsnetwork/calcprob2324/assets/107944476/170f45b5-2260-496a-8781-9d776683e082)

**Comment by daromacarlo** [comment]
![Immagine 2023-11-02 125310](https://github.com/sapienzastudentsnetwork/calcprob2324/assets/135441131/5925d1d8-7baa-4421-a152-6846b6cc6025)

**Comment by Elia-Belli** [comment, marked as answer]
Sappiamo che $\mathbb{P}(Omo)=p$, $\mathbb{P}(Etero)=1-p$ e $\mathbb{P}(SameSex|Etero)=0.5$, inoltre $\mathbb{P}(SameSex|Omo)=1$
- Per Bayes:
$$\mathbb{P}(Omo|SameSex)=\frac{\mathbb{P}(Omo)\cdot \mathbb{P}(SameSex|Omo)}{\mathbb{P}(Omo)\cdot \mathbb{P}(SameSex|Omo)+\mathbb{P}(Etero)\cdot \mathbb{P}(SameSex|Etero)}$$
$\mathbb{P}(Omo|SameSex)= \frac{p\cdot 1}{p\cdot 1+(1-p)\cdot 0.5}=\frac{p}{p+0.5-0.5p}=0.5+\frac{p}{0.5}$
- Per la formula delle probabilità totali:
$$\mathbb{P}(DiffSex)=\mathbb{P}(Etero)\cdot \mathbb{P}(DiffSex|Etero)+ \mathbb{P}(Omo)\cdot \mathbb{P}(DiffSex|Omo)$$
$\mathbb{P}(DiffSex)=(1-p)(1-0.5)+0=0.5- 0.5p$

**Comment by LeoCosta02** [comment]
@Elia-Belli nel punto 2 hai sbagliato il segno di 0.5p viene -0.5p perchè il prodotto è (1-p) (1/2)

**Comment by CiottoloMaggico** [comment]
Analogo agli altri
![image](https://github.com/sapienzastudentsnetwork/calcprob2324/assets/79638284/5bebbb8d-c854-4c21-9aa2-1c76192dd933)

**Comment by CasuFrost** [comment]
![image](https://github.com/sapienzastudentsnetwork/calcprob2324/assets/97389762/2d41bbf6-8218-4182-97db-646881e9be73)
